src/main/java/org/example/lib/ORManager.java

```diff
@@ -44,13 +46,11 @@ static ORManager withPropertiesFrom(String filename) {
         }
 
```

Comment:
In my opinion is better to have separate Builder class for withPropertiesFrom(String filename), where you should have the method, but returning DataSource.